tests/Microsoft.Identity.Test.Unit/PublicApiTests/TelemetryClientTests.cs

```diff
@@ -153,6 +153,7 @@ private void AssertLoggedTelemetry(AuthenticationResult authenticationResult, Ms
             Assert.AreEqual(authenticationResult.AuthenticationResultMetadata.DurationInHttpInMs, eventDetails.Properties[TelemetryConstants.DurationInHttp]);
             Assert.AreEqual(authenticationResult.AuthenticationResultMetadata.DurationInCacheInMs, eventDetails.Properties[TelemetryConstants.DurationInCache]);
             Assert.AreEqual(authenticationResult.AuthenticationResultMetadata.DurationTotalInMs, eventDetails.Properties[TelemetryConstants.Duration]);
+            Assert.AreEqual(typeof(ConfidentialClientApplication).Assembly.GetName().Version.ToString(), eventDetails.Properties[TelemetryConstants.MsalVersion]);
```

Comment:
Use MsalIdHelper.GetMsalVersion() instead? Plus wouldn't the assembly version be different than Msal Version (ex 4.46.2.0 vs 4.46.2), so this should fail?

Reply:
Removed this test as no longer relevant